addon/utils/pretty-percentage.js

```diff
@@ -0,0 +1,10 @@
+export default function prettyPercentage(decimalValue, precision=0) {
```

Comment:
I'm inclined to think we should make a new add-on. Consider [format-currency](https://github.com/PrecisionNutrition/payment-next/blob/f58fd9f575d10e5528660a5cecba15d696966b5c/app/helpers/format-currency.js) in Payment Next. This is also number formatting. I think if you pull this out, and format-currency we have ourselves a bonafide new add-on.